### PR TITLE
Fixed unreasonable reserved resource example values

### PR DESCRIPTION
**Pull request by ghost**
@nekop Please review. This is a follow-up to https://github.com/openshift/openshift-docs/pull/4963.

@openshift/team-documentation PTAL. No rev history needed.

### CODE REVIEW

**Comment by mburke5678** [comment]
@tmorriso-rh LGTM

File: admin_guide/allocating_node_resources.adoc [hunk 1]
```diff
@@ -42,23 +42,23 @@ You can set these in the `*kubeletArguments*` section of the
...
   system-reserved:
-    - "cpu=200m,memory=30G"
+    - "cpu=200m,memory=512Mi"
 ----
 ====
 
 Currently, the `*cpu*` and `*memory*` resource types are supported. For `*cpu*`,
-the resource quantity is specified in units of cores (e.g., 200m, 100Ki, 50M).
-For `*memory*`, it is specified in units of bytes (e.g., 200Ki, 100M, 50Gi).
+the resource quantity is specified in units of cores (e.g., 200m, 0.5Ki, 1M).
```
Comment (nekop) [inline admin_guide/allocating_node_resources.adoc]: For cpu there is no 'Ki' or 'M'. It should be:

> (e.g., 200m, 0.5, 1)

Comment (tmorriso-rh) [reply admin_guide/allocating_node_resources.adoc]: I changed this to (e.g., 200m, 0.5, 1). Please review. Thanks.

**Comment by tmorriso-rh** [comment]
Note to TW: manual cherry pick to 3.2 - 3.4. Use script for 3.5 - 3.9.

**Pull request merged**